README.md

```diff
@@ -1,3 +1,5 @@
 # RHMAP Nagios for Docker
 
+[![Build Status](https://travis-ci.org/feedhenry/nagios-container.svg?branch=master)](https://travis-ci.org/feedhenry/nagios-container)
```

Comment:
Is it also worth adding in a section which outlines the two validation commands being run as part of the travis build? i.e. pep8 plugins/ --ignore E501 --show-source and python -m unittest discover -s plugins/default so that people know how to run this tests locally beforehand.